content/en/docs/installation/cli.md

```diff
@@ -4,27 +4,89 @@ description: Install the notation CLI on Linux, macOS, and Windows
 weight: 1
 ---
 
-## Download the `notation` CLI 
+## Download the `notation` CLI
 
-Download the latest CLI for your platform from the [Releases page](https://github.com/notaryproject/notation/releases).
+Navigate to the [Releases page](https://github.com/notaryproject/notation/releases) and select the latest release of the CLI. The notation CLI binaries for each platform are available under the *Assets* section in the `notation_<version>_<platform>.tar.gz` files.
 
-## Install the CLI for Linux
+For example, in the [v0.12.0-beta1 release](https://github.com/notaryproject/notation/releases/tag/v0.12.0-beta.1) the following CLI binaries are available:
 
-Expand the archive file and add it to your path. For example, the following command expands the archive file directly to `~/bin`.
+* `notation_0.12.0-beta.1_darwin_amd64.tar.gz` for macOS running on Intel processors.
+* `notation_0.12.0-beta.1_darwin_arm64.tar.gz` for macOS running on Apple Silicon processors.
+* `notation_0.12.0-beta.1_linux_amd64.tar.gz` for Linux running on Intel processors.
+* `notation_0.12.0-beta.1_linux_arm64.tar.gz` for Linux running on ARM processors.
+* `notation_0.12.0-beta.1_windows_amd64.zip` for Windows running on Intel processors.
 
-```console
-tar xvzf notation.tar.gz -C ~/bin notation
+## Verify the `notation` CLI checksum
```

Comment:
Is it necessary to showcase how to verify checksum in the installation guide? I think we can note this step is recommended but not a MUST.